core/keyboard_nav/navigation.js

```diff
@@ -567,17 +581,17 @@ Blockly.navigation.disconnectBlocks_ = function() {
  */
 Blockly.navigation.markAtCursor_ = function() {
   var workspace = Blockly.getMainWorkspace();
-  workspace.getMarker().setCurNode(workspace.getCursor().getCurNode());
+  Blockly.navigation.getMarker().setCurNode(workspace.getCursor().getCurNode());
```

Comment:
I'm curious why we go through ``Blockly.navigation`` to get the marker, when we could just use the workspace here.

Reply:
My though was that for now we only want to be getting the local marker in navigation and Blockly.navigation.getMarker is nicer than Blockly.getMainWorkspace().getMarker(Blockly.navigation.MARKER_NAME) but doesn't really matter either way.

Reply:
Fair enough, in that case I would just do: 
```
Blockly.navigation.getMarker().setCurNode(Blockly.getMainWorkspace().getCursor().getCurNode());
```
There's no need for the workspace variable.